charts/camunda-platform/values.yaml

```diff
@@ -179,9 +179,9 @@ global:
         #   port: "8443"
       #  defines the endpoint of Keycloak which varies between Keycloak versions.
       ## @param global.identity.keycloak.contextPath In Keycloak v16.x.x it's hard-coded as '/auth', but in v19.x.x it's '/'.
-      contextPath: "/auth"
+      contextPath: "/auth/"
       ## @param global.identity.keycloak.realm defines Keycloak realm path used for Camunda.
-      realm: "/realms/camunda-platform"
+      realm: "realms/camunda-platform"
```

Comment:
I'm sure this will break an app library somewhere :jack_o_lantern: 
Let's just add the extra slash in the Ingress.